### ISSUE TITLE
Thruk 3.0 display issue

### ISSUE THREAD

**Issue by exodus1984** [sni/Thruk#1214]
The system information field overlaps other text fields.

Thruk 3.0

Visible wherever system info is available on the host.

**Screenshots**
![error2](https://user-images.githubusercontent.com/17269638/202397224-ce72ad22-078e-4c0e-9d1a-a38704020283.png)
![error1](https://user-images.githubusercontent.com/17269638/202397233-fec8e085-35e6-4ab7-a2b7-0d2ae5b3d742.png)

**Comment by sni** [comment]
That's the `alt` for the host icon image. This basically means the images is missing.

**Comment by sni** [comment]
No, i don't think so. Those icon images are not shipped with Thruk.
